app/src/androidTest/java/org/mozilla/fenix/ui/BookmarksTest.kt

```diff
@@ -43,57 +43,155 @@ class BookmarksTest {
     @After
     fun tearDown() {
         mockWebServer.shutdown()
+        val bookmarksStorage = activityTestRule.activity?.bookmarkStorage
```

Comment:
Perhaps a comment saying that the bookmarks are deleted after each test?

Reply:
Done, updating the branch right now